--- configure	2024-07-12 09:33:31.349200777 +0100
+++ configure	2024-07-12 09:42:23.849197619 +0100
@@ -24741,8 +24741,8 @@
     # additional preprocessor flags for NumPy
     python_out=`cat <<EOD | ${PYTHON} - 2>/dev/null
 import sys
-import numpy.lib.utils as npyutil
-sys.stdout.write(' -I' + npyutil.get_include())
+import numpy
+sys.stdout.write(' -I' + numpy.get_include())
 EOD`
     if test $? -ne 0; then :
 
